### PR TITLE
drivers*: add drivers with saul support in the doxygen drivers_saul group

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR is a follow-up of #9328 that put a doxygen dependency of drivers with saul support to the drivers_saul group.
This way all drivers with SAUL support can be reached directly when browsing the SAUL module documentation.

### Issues/PRs references

follow-up of #9328

### CODE REVIEW

**Comment by gebart** [comment]
I like this idea!
When you use `@ingroup` twice the page is available from two places in the ToC? Nice feature

File: drivers/include/lsm6dsl.h [hunk 1]
```diff
@@ -10,8 +10,10 @@
 /**
  * @defgroup    drivers_lsm6dsl LSM6DSL 3D accelerometer/gyroscope
  * @ingroup     drivers_sensors
- * @brief       Device driver for the LSM6DSL 3D accelerometer/gyroscope
+ * @ingroup     drivers_saul
+ * @brief       Device drive for the LSM6DSL 3D accelerometer/gyroscope
```
Comment (jia200x) [inline drivers/include/lsm6dsl.h]: s/drive/driver

**Comment by jia200x** [comment]
looks good. There's only a minor issue I just posted

File: drivers/include/lsm6dsl.h (hunk 1 above)
Comment (aabadie) [reply drivers/include/lsm6dsl.h]: Fixed and directly squashed

**Pull request merged**